### ISSUE TITLE
Single file downloads are always overwitten

### ISSUE THREAD

**Issue by SheffSix** [HaveIBeenPwned/PwnedPasswordsDownloader#23]
It seems that, if downloading to a single file, the file is always overwritten, regardless of --overwrite or -o being used or not.

Is this intentional for single files?

Using PopOS 22.04 and .net runtime from the default PopOS apt sources.

**Comment by SheffSix** [comment]
Line 121 is 
```c#
if (File.Exists(settings.OutputFile))
```

The script always appends ".txt" to single files, so the check needs to take this into account. OR the script shouldn't be add file extensions if the user hasn't specified one.

```c#
if (File.Exists($"{settings.OutputFile}.txt"))
```

(This is an assumption: I don't know C# and have no way to test so I'm not going to do a pull request myself)

**Comment by stebet** [comment]
Should be fixed in #26. Thank you @SheffSix!
